install/tests/test_installtion.py

```diff
@@ -1,49 +1,42 @@
 from .base_test import BaseTest
-from parameterized import parameterized
-import os
+import uuid
 
 
-class Test_instaltion(BaseTest):
+class TestInstallationInDocker(BaseTest):
     def setUp(self):
-        pass
+        self.CONTAINER_NAME = str(uuid.uuid4()).replace("-", "")[:10]
+        self.info(
+            "Install container jumpscale from {} branch in {} os type".format(self.get_js_branch(), self.get_os_type()))
+        output, error = self.jumpscale_installation("container-install", "-n {}".format(self.CONTAINER_NAME))
```

Comment:
why we install jumpscale every testcase ? we just need it one time

Reply:
actually, we are testing the installation itself with a specific action that is why we need to install/remove in setup/teardown. In general, the test case should be independent of each other. So, if one of the test cases ruin the jsx installation, we will be sure it won't affect the other test cases.